website/src/components/casestudy/casestudy-list.tsx

```diff
@@ -1,37 +1,94 @@
 import React from "react"
 import common from "../../css/common.module.css"
 import styles from "./casestudy.module.css"
+import Translate from "@docusaurus/Translate"
 
 const caseStudies = [
   {
     name: "Adjacent Cores for Operators",
-    description:
-      "Mobile Network Operators face the challenge of scalability on a daily basis. Highly complex networks that limit the number of use-cases and additional complexity they can take on is the norm.",
+    description: (
+      <Translate
+        id="home.casestudy.adjacentcores.description"
+        description="Description for the adjacent cores for operators case study"
```

Comment:
The description is supposed to be repeated both here and in the translation (i.e. website/i18n/jp/code.json)?

Reply:
Yes. When you have wrapped the text that should be translatable in the `<Translate>` component, you run `docusaurus write-translations` and the translation files (`/i18n/jp/**/*.json`) are automatically generated. The description are there in the json file for you to understand what the texts refer to.